examples/user_interfaces/embedding_in_qt5.py

```diff
@@ -117,7 +117,7 @@ def closeEvent(self, ce):
         self.fileQuit()
 
     def about(self):
-        QtGui.QMessageBox.about(self, "About",
+        QtWidgets.QMessageBox.about(self, "About",
                                 """embedding_in_qt5.py example
```

Comment:
Ah, sorry about that. Commit updated. I also removed the now unused import of `QtGui`.